bears/configfiles/DockerfileLintBear.py

```diff
@@ -44,4 +44,4 @@ def process_output(self, output, filename, file):
                     message=issue["message"],
                     file=filename,
                     severity=self.severity_map[issue["level"]],
-                    line=issue["line"])
+                    line=issue.get("line", None))
```

Comment:
default is: dict.get(key, default=None)

so wouldnt it make sense not to specify "None" there if its default?